src/postcapture.rs

```diff
@@ -10,21 +10,31 @@ use crate::{
     historymodel::{ModelNotifier, RowData},
 };
 
+/// Trait for the post capture actions.
 pub trait PostCaptureAction {
-    fn handle(&self, model_notifier: ModelNotifier, conn: &SqliteConnection, pixbuf: Pixbuf);
-}
+    /// The name of the post capture action.
+    fn name(&self) -> String;
+
+    /// Short description of the post capture action.
+    fn description(&self) -> String;
 
-pub fn current_action() -> &'static dyn PostCaptureAction {
-    // FIXME: Eventually this should do more than just this, but we'll get there
-    &SaveAndCopy
+    /// Gets called when executing the post capture action.
+    fn handle(&self, model_notifier: ModelNotifier, conn: &SqliteConnection, pixbuf: &mut Pixbuf);
 }
 
-/// This struct represents an action that saves the screenshot to disk and then copies it into
-/// the user's clipboard
-struct SaveAndCopy;
+/// This struct represents the action of saving the pixbuf to disk.
+pub struct SaveToDisk;
+
+impl PostCaptureAction for SaveToDisk {
+    fn name(&self) -> String {
+        "Save to disk".to_owned()
+    }
+
+    fn description(&self) -> String {
+        "Saves the screenshot to the Harddrive".to_owned()
```

Comment:
This is inaccurate as the user may be using an SSD. (Also words don't get capitalised this way in English)
```suggestion
        "Saves the screenshot to the disk".to_owned()
```